core/src/main/java/org/infinispan/filter/CompositeKeyValueFilter.java

```diff
@@ -34,6 +35,13 @@ public boolean accept(K key, V value, Metadata metadata) {
       return true;
    }
 
+   @Inject
+   protected void injectDependencies(ComponentRegistry cr) {
+      for (KeyValueFilter<? super K, ? super V> f : filters) {
+         cr.wireDependencies(f);
+      }
+   }
+
    public static class Externalizer extends AbstractExternalizer<CompositeKeyValueFilter> {
```

Comment:
Technically this Externalizer can use the UnsignedNumeric as well, sorry about missing this originally.